templates/event.php

```diff
@@ -79,13 +79,30 @@
 		<h2><?php esc_html_e( 'Contributors', 'gp-translation-events' ); ?></h2>
 		<ul>
 			<?php foreach ( $contributors as $contributor ) : ?>
-			<li class="event-contributor" title="<?php echo esc_html( implode( ', ', $contributor->locales ) ); ?>"
+			<li class="event-contributor" title="<?php echo esc_html( implode( ', ', $contributor->locales ) ); ?>">
 				<a href="<?php echo esc_url( get_author_posts_url( $contributor->ID ) ); ?>"><?php echo get_avatar( $contributor->ID, 48 ); ?></a>
 				<a href="<?php echo esc_url( get_author_posts_url( $contributor->ID ) ); ?>"><?php echo esc_html( get_the_author_meta( 'display_name', $contributor->ID ) ); ?></a>
 			</li>
 		<?php endforeach; ?>
 		</ul>
 	</div>
+	<div class="event-projects">
+		<h2><?php esc_html_e( 'Projects', 'gp-translation-events' ); ?></h2>
+		<ul>
+			<?php foreach ( $projects as $project_name => $row ) : ?>
+			<li class="event-project" title="<?php echo esc_html( str_replace( ',', ', ', $row->locales ) ); ?>">
+				<a href="<?php echo esc_url( gp_url_project( $row->project ) ); ?>"><?php echo esc_html( $project_name ); ?></a> <small> to
+				<?php
+				foreach ( explode( ',', $row->locales ) as $_locale ) {
+					$_locale = \GP_Locales::by_slug( $_locale );
+					?>
+					<a href="<?php echo esc_url( gp_url_project_locale( $row->project, $_locale, 'default' ) ); ?>"><?php echo esc_html( $_locale->english_name ); ?></a>
+				<?php } ?>
+				by <?php echo esc_html( $row->users ); ?> contributors</small>
```

Comment:
```suggestion
				by 
				<?php
				// translators: %d: Number of contributors.
				printf( _n( '%d contributor', '%d contributors', $row->users, 'gp-translation-events' ), esc_html( $row->users ) );
				?>
```

Reply:
Just a tiny suggestion for singular and plurals.

Reply:
Thanks for the suggestion, we should just use the `esc_html()` on the outside and `sprintf` in the inside to make sure all of it is escaped.